canonicalwebteam/discourse/parsers/base_parser.py

```diff
@@ -601,7 +601,7 @@ def _replace_notifications(self, soup):
         Becomes:
 
             <div class="p-notification">
-                <div class="p-notification__response">
+                <div class="p-notification__content">
```

Comment:
You need `p-notification__content` and `p-notification__message` inside it:

```html
<div class="p-notification--information">
  <div class="p-notification__content">
    <h5 class="p-notification__title">Optional title</h5>
    <p class="p-notification__message">Anyone with access can view your invited users.</p>
  </div>
</div>
```

So if the `<p>` comes from discourse you probably need to add `p-notification__message` class to it.